iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/agent/task/subtask/connector/PipeConnectorSubtask.java

```diff
@@ -199,7 +199,13 @@ public void close() {
 
     isClosed.set(true);
     try {
+      final long startTime = System.currentTimeMillis();
       outputPipeConnector.close();
+      LOGGER.info(
+          "Pipe: connector subtask {} close {} within {} ms",
+          taskID,
+          outputPipeConnector.getClass().getName(),
```

Comment:
```suggestion
          outputPipeConnector.getClass().getName(),
```

ID is better than class name?

Reply:
the Java object ID?

Reply:
taskID